cmd/containerd/command/main.go

```diff
@@ -130,6 +131,13 @@ can be used and modified as necessary as a custom configuration.`
 			return err
 		}
 
+		// Initialize openTelemetry tracing
+		log.G(ctx).Info("starting openTelemetry")
```

Comment:
These seem like debug messages.

Reply:
feels like it's at the wrong scope and log level..  suggest moving into initopentelemetry not sure if you need to log even debug If the config isn't on.. you'll have that from info..